setup.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup, find_packages
-from pkg_resources import require, DistributionNotFound
-import pages
```

Comment:
I don't see how __import__('pages') is nicer to the eye that import pages (it is not), __import__ is used to import something that you don't know the name in advance... So this is unnecessary here.